scalameta/parsers/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -438,6 +438,65 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     if (!token.is[StatSeqEnd])
       acceptStatSep()
 
+/* ------------- MODIFIER VALIDATOR --------------------------------------- */
+
+
```

Comment:
Can we remove one of the newlines here to be consistent with other `/* ---- ... ---- */` sections?